PWGGA/GammaConv/macros/AddTask_GammaConvDalitzV1_pp.C

```diff
@@ -1,6 +1,6 @@
 
 void AddTask_GammaConvDalitzV1_pp(  Int_t trainConfig = 1,  //change different set of cuts
-                                    Bool_t isMC   = kFALSE, //run MC
+                                    Bool_t isMC   = kFALSE, //0 Exp data, 1 MC data, 2 JJ MC for improves.
```

Comment:
The comment makes no sense if the variable is a boolean. Do you want to change this to an integer?